### PR TITLE
add signal handler for fpe, bus error

### PR DESCRIPTION
**Pull request by szha**
## Description ##
add signal handler for fpe, bus error

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at https://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: include/mxnet/runtime/c_runtime_api.h [hunk 1]
```diff
@@ -177,4 +178,43 @@ MXNET_DLL int MXNetObjectTypeKey2Index(const char* type_key, unsigned* out_tinde
 #ifdef __cplusplus
 }  // extern "C"
 #endif
+
```
Comment (samskalicky) [inline include/mxnet/runtime/c_runtime_api.h:181]: Can we copy over the comment too?

Comment (samskalicky) [reply include/mxnet/runtime/c_runtime_api.h:181]: thanks, only one nit: should it be MX_API_BEGIN instead of API_BEGIN?

Comment (szha) [reply include/mxnet/runtime/c_runtime_api.h:181]: https://github.com/apache/incubator-mxnet/blob/master/src/c_api/c_api_common.h#L45-L55

Comment (samskalicky) [reply include/mxnet/runtime/c_runtime_api.h:181]: is this useful or should we get rid of this indirection?

Comment (szha) [reply include/mxnet/runtime/c_runtime_api.h:181]: good point, removed. thanks for bringing this up

Comment (szha) [reply include/mxnet/runtime/c_runtime_api.h:181]: OK, there's external dependency on this ... https://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-gpu/detail/PR-18956/7/pipeline
I will revert this change for now.

**Pull request merged**